src/Reveal.Sdk.Dom.Tests/Visualizations/SingleValueLabelsVisualizationBaseFixture.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Visualizations;
+using Reveal.Sdk.Dom.Visualizations.Settings;
+using Reveal.Sdk.Dom.Visualizations.VisualizationSpecs;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations
+{
+    public class SingleValueLabelsVisualizationBaseFixture
+    {
+        [Theory]
+        [InlineData("testTitle")]
+        [InlineData(null)]
+        public void Constructor_SetTitleAndDataSourceItem_WhenConstructed(String title)
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+
+            // Act
+            var visualization = new TestSingleValueLabelsVisualization(title, dataSourceItem);
+
+            // Assert
+            Assert.Equal(title, visualization.Title);
+            Assert.Equal(dataSourceItem, visualization.DataDefinition.DataSourceItem);
+            Assert.IsType<SingleValueLabelsVisualizationDataSpec>(visualization.VisualizationDataSpec);
+        }
+
+        [Fact]
+        public void GetLabels_ReturnsVisualizationSpecRows_IfDataSpecIsSingleValueLabelsVisDataSpec()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new TestSingleValueLabelsVisualization("testTitle", dataSourceItem);
+            var visSpec = new SingleValueLabelsVisualizationDataSpec()
+            {
+                Rows = new List<DimensionColumn>
+                {
+                    new(),
+                    new(),
+                }
+            };
+            visualization.VisualizationDataSpec = visSpec;
+
+            // Act
+            var labels = visualization.Labels;
+
+            // Assert
+            Assert.Equal(visSpec.Rows, labels);
+        }
+
+        [Fact]
+        public void GetLabels_ReturnsEmptyList_IfDataSpecIsNotSingleValueLabelsVisDataSpec()
+        {
+            // Arrange
+            var dataSourceItem = new DataSourceItem();
+            var visualization = new TestSingleValueLabelsVisualization("testTitle", dataSourceItem);
+            var visSpec = new CategoryVisualizationDataSpec();
+            visualization.VisualizationDataSpec = visSpec;
+
+            // Act
+            var labels = visualization.Labels;
+
+            // Assert
+            Assert.Empty(labels);
```

Comment:
I think we should add an assertion to verify that the returned list is not null, even if empty